packages/react/src/auto/hooks/useModelActionsForTable.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from "react";
+import deepEqual from "react-fast-compare";
+import { humanizeCamelCase } from "../../utils.js";
+
+export type ModelActionDetails = {
+  apiIdentifier: string;
+  operationName: string;
+  isDeleter: boolean;
+  isBulk: boolean;
+  acceptsModelInput: boolean;
+  variables: any;
+};
+
+/**
+ * Returns a list of ModelActionDetail objects for given model
+ */
+const useModelActionsForTable = (props: { model: any }) => {
+  const modelActions: any[] = [];
```

Comment:
Nit:
```ts
const modelActions: ModelActionDetails[] = [];
```

Btw forgot to remove console.log in line 20?

Reply:
Updated to be `ModelActionDetails` and removed the leftover log